### PR TITLE
Connectivity

### PR DESCRIPTION
**Pull request by amandadumi**
This pull request accomplishes three things:
1) updating the butane molecule
2) creates a connectivity matrix for xyz files to expand the file types accepted for many of the representations.
3) introduces the dependency on qc.elemental for covalent radii and atom number.

What still needs done:
1) update requirements

### CODE REVIEW

**Comment by dlf57** [comment]
I think if you update the requirement in setup.py it will at least fix the current issue Travis is having.

**Comment by dlf57** [comment]
The requirements.txt is for the documentation builds

**Comment by amandadumi** [comment]
hahah i'll fix that, thank you!

**Comment by dlf57** [comment]
Np! It's really confusing having that there as well!

**Comment by dlf57** [comment]
Everything looks like its passing! I will take a more in depth look into the changes tomorrow

File: chemreps/utils/molecule.py [hunk 1]
```diff
@@ -214,3 +195,50 @@ def import_cclib(self, fname):
...
+        BM: ndarray
+            each entry (ij) of bond matrix indicates whether one atom i is
+            bonded to atom j. entry is 1 if bonded 0 if not.
+        """
+        #TODO: make this more memory efficient by ordering i,j in code when needed. 
+        temp = []
+        for i in range(self.n_atom):
+            for j in range(i+1, self.n_atom):
+                    if self.bond(i,j):
+                        temp.append([i+1,j+1])
```
[the remark this answers is not in the document]
Comment (shivupa) [reply chemreps/utils/molecule.py]: Could you leave a note that if we are trying to access the connectivity matrix we should be checking cases where i<j. So `any(np.equal(connect_mat_true,[1,2]).all(1))` will be true but not `any(np.equal(connect_mat_true,[2,1]).all(1))` if atoms 1 and 2 are bonded

[the remark this answers is not in the document]
Comment (amandadumi) [reply chemreps/utils/molecule.py]: added  comments!

**Comment by amandadumi** [comment]
QCElemental has been added to the dependencies!

File: README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ chemreps requires:
 - Python (>=3.6)
 - NumPy (>=1.12)
 - cclib (>=1.5)
+  QCElemental
```
Comment (dlf57) [inline README.md]: Sorry to be too nitpicky but the formatting is off.  Currently it shows up as ` - cclib (>=1.5) QCElemental`

Comment (amandadumi) [reply README.md]: No that's a great point! I couldn't see the dashes grayed out to match the background in my text editor so I didn't see them last night! Should be fixed now. Thanks.

**Comment by dlf57** [comment]
Thanks! Everything looks to be working so I'll merge!

**Pull request merged**